### ISSUE TITLE
Update ETH Staking Keys docs to reflect `lattice-eth2-utils`

### ISSUE THREAD

**Issue by alex-miller-0** [GridPlus/gridplus-sdk#510]
In #508 we removed bespoke test functionality related to BLS in favor of using `lattice-eth2-utils`.

I forgot to update the docs, which are now out of date: https://gridplus.github.io/gridplus-sdk/tutorials/ethDeposits

We should update the docs and maybe ship a fix version so that people don't get confused looking at that.

**Comment by alex-miller-0** [comment]
Perhaps we should actually move this to a new section... it doesn't really belong with the other sections, which describe how to do things with the SDK. Here we should explain how to do things with another tool, which happens to itself use the SDK...

**Comment by douglance** [comment]
We _could_ move all the docs to `docs.gridplus.io`. It's not ideal, but we can ship markdown to that service, so I bet we could migrate over in a few hours. The biggest issue with that would be ongoing maintenance. 

Perhaps we move all the manual docs to the `docs.gridplus.io`, then keep the automated docs as a reference? Or just remove the automation entirely?

**Comment by alex-miller-0** [comment]
What do you mean by manual vs automated docs?

**Comment by douglance** [comment]
Everything under the API heading is auto-generated.

**Comment by alex-miller-0** [comment]
Gotcha.

Yeah so a few points here:

* There are basic SDK tutorial docs like the [signing pages](https://gridplus.github.io/gridplus-sdk/signing). I think these make sense to stay in the SDK docs page with the autogenerated docs
* I created a separate bucket called "Tutorials" for patterns that might be used by integrations. These tutorials include [calldata decoding](https://gridplus.github.io/gridplus-sdk/tutorials/calldataDecoding) and [address tags](https://gridplus.github.io/gridplus-sdk/tutorials/addressTags)
* In the Tutorials section we also have [ETH Staking Keys](https://gridplus.github.io/gridplus-sdk/tutorials/ethDeposits), which was meant to be a "higher order" doc on how to do ETH2 stuff. However, all of this has gotten moved to `lattice-eth2-utils`. I'm not sure where to put these docs. One option is to create a new section for "using external utils" or something and put the ETH2 stuff in there. Another option is to add a very short description of `lattice-eth2-utils` and link out from the SDK docs, then to add an explanation of the CLI actions in the official GP docs. In the CLI docs we will also link out to `lattice-eth2-utils` for developers.
* Basically we need two things: 1) How do we show *developers* where to go to integrate the Lattice for ETH2 functionality (note that there probably isn't going to be a lot of ETH2 integrations anytime soon). 2) How do we show *users* what ETH2 functionality their Lattice can do? This would need to include the CLI because that's going to be the easiest user facing interface for now. We can look into adding a manager page but it will be a bit complicated and anyone generating keystores is probably at the command line anyway.

**Comment by alex-miller-0** [comment]
I agree. I guess we can just link to `lattice-eth2-utils` in a note for developers
